sdks/go/data/fs/handle.go

```diff
@@ -53,7 +48,7 @@ func (lh handle) rListDescendants(
 	[]*model.DirEntry,
 	error,
 ) {
-	childFileInfos, err := lh.ioUtil.ReadDir(path)
+	childFileInfos, err := ioutil.ReadDir(path)
```

Comment:
Just an FYI if you hadn't seen, `io/ioutil` is being deprecated in go 1.16. This'll be moved to `os.ReadDir`

Reply:
Nice. I gave a quick shot at upgrading to 1.16 but tests weren't passing so whoever gets to it first will have to look into it; assuming won't be too bad. We should ditch `io/ioutil` when we do.